macros/models/contracts/find_self_destruct_contracts.sql

```diff
@@ -63,7 +63,7 @@ FROM (
       AND sds.type = 'suicide'
       AND sds.address IS NOT NULL
       {% if is_incremental() %}
-      and sds.block_time >= date_trunc('day', now() - interval '7' day)
+      AND {{ incremental_predicate('sds.block_time') }}
       and cr.address NOT IN (SELECT contract_address FROM {{this}} th WHERE th.blockchain = '{{chain}}' ) --ensure no duplicates
```

Comment:
same question here, on the same filter condition

Reply:
actually, we should be able to remove this one